src/transformers/models/layoutlmv3/tokenization_layoutlmv3_fast.py

```diff
@@ -640,18 +639,23 @@ def _batch_encode_plus(
                 else:
                     original_index = batch_index
                 labels_example = []
+                previous_token_empty = False
                 for id, offset, word_id in zip(
                     sanitized_tokens["input_ids"][batch_index],
                     sanitized_tokens["offset_mapping"][batch_index],
                     sanitized_encodings[batch_index].word_ids,
                 ):
                     if word_id is not None:
                         if self.only_label_first_subword:
-                            if offset[0] == 0:
+                            if offset[0] == 0 and not previous_token_empty:
```

Comment:
Wouldn't 
```suggestion
                            if offset[0] == 0 and offsets[1] != 0:
```
be enough ? Removing the need for additional state ?

Reply:
It is not equivalent because the logic relies on the previous token being empty. A quick example:
`"phone 0000000000000000"` is parsed into
tokens: `["Ġphone", "Ġ",    "0000000000000000"]`
offsets: `[(0, 5),   (0, 0), (0, 16)]`
Notice the "empty" token introduced.

But in the context of labeling on tokens at the beginning of words (`only_label_first_subword` option), we expect the labels for tokens to be
labels: ` [1,        2,      -100]`
because token `0000000000000000` is the continuation of the second word.

If I'm correct, your change would label the sequence the following:
labels: ` [1,        -100,   2]`

Reply:
Thanks for the explanation !